client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import FormToggle from 'components/forms/form-toggle';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { isJetpackModuleActive } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { updateSettings } from 'state/jetpack/settings/actions';
+
+const MediaSettings = ( {
+	fields,
+	handleToggle,
+	onChangeField,
+	isRequestingSettings,
+	isSavingSettings,
+	siteId,
+	carouselActive,
+	translate
+} ) => {
+	const labelClassName = isSavingSettings || ! carouselActive ? 'is-disabled' : null;
+
+	return (
+		<Card className="media-settings site-settings">
+			<FormFieldset>
+				<div className="media-settings__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/photon' } >
+							{ translate( 'Learn more about Photon' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+				<JetpackModuleToggle
+					siteId={ siteId }
+					moduleSlug="photon"
+					label={ translate( 'Speed up your images and photos with Photon.' ) }
+					description="Enabling Photon is required to use Tiled Galleries."
+					disabled={ isRequestingSettings || isSavingSettings }
+					/>
+			</FormFieldset>
+			<FormFieldset className="media-settings__formfieldset has-divider is-top-only">
+				<div className="media-settings__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/carousel' } >
+							{ translate( 'Learn more about Carousel' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+				<JetpackModuleToggle
+					siteId={ siteId }
+					moduleSlug="carousel"
+					label={ translate( 'Transform standard image galleries into full-screen slideshows.' ) }
+					disabled={ isRequestingSettings || isSavingSettings }
+					/>
+				<div className="media-settings__module-settings is-indented">
```

Comment:
As #11240 suggests, we no longer need to use `is-indented`. This can also be cleaned up from the selector in `media-settings/style.scss` to simplify it.